rules/sinks/third_parties/sdk/amplitude/java.yaml

```diff
@@ -9,5 +9,29 @@ sinks:
     domains:
       - "amplitude.com"
     patterns:
-      - "(?i)(com[.]amplitude|com[.]rudderstack[.]android[.]amplitude|com[.]rudderstack[.]android[.]integration[.]amplitude).*"
+      - "(?i)(com[.]rudderstack[.]android[.]amplitude|com[.]rudderstack[.]android[.]integration[.]amplitude).*"
+    tags:
+
+  - id: ThirdParties.SDK.Amplitude.Analytics
+    name: Amplitude Analytics
+    domains:
+      - "amplitude.com"
+    patterns:
+      - "(?i)(com[.]amplitude[:][analytics]).*"
```

Comment:
Is the `:` present in `[:]` deliberate ?

Same comment for below rules

Reply:
yes it's deliberate, we used to import something like  'com.amplitude:experiment-android-client:4.16.0' and similar for others

Reply:
Are you referring the gradle dependency declaration?

Reply:
The gradle import is not what we considered for writing sink rule, we look for the `import statement in source code`, better to update the following patterns to `.` instead of `[:]` so that it matches any character

Reply:
okay, updated